src/components/features/main/tree/RegisterSchedulingOptionsModalComponent.tsx

```diff
@@ -47,7 +48,13 @@ export const RegisterSchedulingOptionsModalComponent = (props: Props) => {
   return (
     <ModalComponent isOpen={props.showsModal} onClose={props.handleClose} title='TODOを予定にする際のオプション'>
       <form className='mt-3' id='register-scheduling-options-form' onSubmit={handleSubmit}>
-        <div className='ml-8'>作業する時間帯</div>
+        <div className='flex items-center'>
+          <div className='ml-8'>作業する時間帯</div>
+          <InfoIconComponent
+            content='左側の項目に「作業開始時間」、右側の項目に「作業終了時間」を入力してください。'
```

Comment:
「作業する時間帯とは、スケジュール作成時にTODOを挿入できるかどうかを判定する時間帯である」という意味合いの説明をユーザーに分かりやすいように入れたいと思っていてこのチケットを作成しました！

↓の修正内容でよければ、`Commit suggestion`ボタンで取り込んでしまってください。
@kzk27 さん（dayduleライトユーザー）目線で、↓だと意味がわからないという場合は再検討しましょう！

```suggestion
            content='「作業する時間帯」とは「あなたがTODOを実行できる時間帯」のことです。'
```

すみません 🙇 
このチケットの背景をきちんとリファインメントで記載していなかったのが悪いです、、、。